templates/src/app/graphql/datasources/AuthAPI.js

```diff
@@ -0,0 +1,20 @@
+const { RESTDataSource } = require('apollo-datasource-rest');
+
+class AuthAPI extends RESTDataSource {
+    constructor() {
+        super();
+        this.baseURL = 'http://localhost:3000';

```

Comment:
Why is this specifying localhost and a static port?

Reply:
Didn't find the constant variable to the baseURL yesterday. So l leave it there temporarily. I will replace it today. Thanks.

Reply:
add env variable GRAPHQL_BASE_URL to Kubernetes and replaced hared code with env variable.